docs/app/Examples/elements/List/Content/ListExampleDescription.js

```diff
@@ -4,14 +4,14 @@ import { List } from 'semantic-ui-react'
 const ListExampleDescription = () => (
   <List>
     <List.Item>
-      <List.Icon name='map marker' />
+      <List.Icon name='marker' />
```

Comment:
This was wrong in core docs as well.  There is on a `map` or `marker` icon, not a `map marker`.  The prop type checker is sooo helpful for these :D 

![image](https://cloud.githubusercontent.com/assets/5067638/20916657/b24d2992-bb41-11e6-99dc-0f42b9b5eb7c.png)